custom_components/elkbledom/__init__.py

```diff
@@ -9,17 +10,24 @@
 import logging
 
 LOGGER = logging.getLogger(__name__)
-PLATFORMS = ["light"]
+
+PLATFORMS: list[Platform] = [
+    Platform.LIGHT,
+    Platform.NUMBER,
+]
 
 async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     """Set up ElkBLEDOM from a config entry."""
-    reset = entry.options.get(CONF_RESET, None) or entry.data.get(CONF_RESET, None)
-    delay = entry.options.get(CONF_DELAY, None) or entry.data.get(CONF_DELAY, None)
-    LOGGER.debug("Config Reset data: %s and config delay data: %s", reset, delay)
+    reset = entry.options.get(
+        CONF_RESET, None) or entry.data.get(CONF_RESET, None)
+    delay = entry.options.get(
+        CONF_DELAY, None) or entry.data.get(CONF_DELAY, None)
+    mac = entry.options.get(CONF_MAC, None) or entry.data.get(CONF_MAC, None)
+    LOGGER.debug("Config: Reset: %s, Delay: %s, Mac: %s", reset, delay, mac)
```

Comment:
These changes only to show the mac address in debug logs, I found it useful so I added it